docs/public-networks/reference/cli/subcommands.md

```diff
@@ -57,6 +57,8 @@ You can specify the starting index of the block range to import with `--start-bl
 
 You can specify the ending index (exclusive) of the block range to import with `--end-block`. If omitted, all blocks after the start block will be imported.
 
+You can specify multiple `--from` arguments. This can be useful when blocks have been exported over time to multiple files. If multiple files are provided they will be read in the order specified in the command.
```

Comment:
```suggestion
You can specify the ending index (exclusive) of the block range to import with `--end-block`. If omitted, all blocks after the start block are imported.

You can specify multiple `--from` arguments. This can be useful when blocks have been exported over time to multiple files. If multiple files are provided they are read in the order specified in the command.
```